changelogs/current.yaml

```diff
@@ -9,6 +9,9 @@ behavior_changes:
 
 minor_behavior_changes:
 # *Changes that may cause incompatibilities for some users, but should not for most*
+- area: tls
+  change: |
+    added support for intermediate CA as trusted ca.
```

Comment:
Please elaborate more on what change the user (of envoy) will see: which certs would previously be untrusted but now are trusted? And please link to the proto config field where the trusted CA is configured.

Reply:
Got it, will do that.